### ISSUE TITLE
File upload against disk storage fails if filename isn't URL encoded

### ISSUE THREAD

**Issue by hygt** [BlueBrain/nexus#664]
```
2019-06-14 09:04:12 ERROR [%PARSER_ERROR[traceToken]] c.e.b.nexus.kg.routes.Routes$ - Exception caught during routes processing
akka.http.scaladsl.model.IllegalUriException: Illegal URI reference: Invalid input ' ', expected '/', 'EOI', '#', '?' or pchar (line 1, column 85): file:///opt/binaries/02cfb59c-79c3-419d-a96c-2c143ec52c26/6/d/f/f/2/8/0/6/Screenshot 2019-06-14 at 10.13.31.png
```

**Comment by umbreak** [comment]
I cannot reproduce the issue either with postman or curl:
```
curl -XPOST -H "Authorization: Bearer $TOKEN" -F "file=@/Users/dmontero/Desktop/Screenshot 2019-06-14 at 10.13.31.png;type=image/png" "http://dev.nexus.ocp.bbp.epfl.ch/v1/files/myorg/myproject"
```

works for me

**Comment by hygt** [comment]
Fixed by BlueBrain/nexus-kg#679
